pack/cap_encounter.json

```diff
@@ -26,8 +42,37 @@
 		"scheme": 1,
 		"set_code": "captain_america_nemesis",
 		"set_position": 2,
-		"text": "Quickstrike.\nWhile Baron Zemo is engaged with you, you cannot thwart.\n<i>(Captain America's nemesis minion.)</i>",
-		"traits": "Assassin. Elite. Mercenary.",
+		"text": "Quickstrike.\nWhile Baron Zemo is engaged with you, you cannot thwart.\n(Captain America's nemesis minion.)",
+		"traits": "HYDRA.ELITE.",
 		"type_code": "minion"
-	}
-]
\ No newline at end of file
+	},
+	{
+		"attack": 2,
+		"boost": 1,
+		"code": "03029",
+		"faction_code": "encounter",
+		"health": 4,
+		"name": "Hydra Soldier",
+		"pack_code": "cap",
+		"position": 29,
+		"quantity": 2,
+		"scheme": 1,
+		"set_code": "captain_america_nemesis",
+		"set_position": 3,
+		"text": "Guard. (While this minion is engaged with you, you cannot attack the villain.)\n<b>When Defeated</b>: Deal the engaged player an encounter card.",
```

Comment:
Would be good to put italics around the rule reminder text (`<i>(While this minion...</i>`)

Reply:
Updated a new commit